src/game_player.cpp

```diff
@@ -613,10 +613,12 @@ bool Game_Player::Move(int dir) {
 		if (!terrain->on_damage_se || (terrain->on_damage_se && (terrain->damage > 0))) {
 			Main_Data::game_system->SePlay(terrain->footstep);
```

Comment:
this seems to match. Can you quickly check that the condition when the SE plays is correct?